docs/reference/components/sources/kubernetes_logs.cue

```diff
@@ -77,6 +77,22 @@ components: sources: kubernetes_logs: {
 							default: "kubernetes.container_name"
 						}
 					}
+					pod_ip: {
+						common: false
+						description: "Event field for Pod IP."
```

Comment:
I think descriptions could afford to be a bit more descriptive. Maybe something like this:

"The name of the event field displaying the Pod's IP address."

Reply:
I've clarified these slightly, but if we change these to make them more verbose we probably want to change the others as well. This is terse, agreed, but it follows the previously established pattern in the file. I'd personally prefer to get these in and then follow up syntactical changes with a secondary PR across the full file.

Reply:
I'm pretty sure I tried something in the form of `The name of the event field displaying the Pod's IP address.` first, and then after some corrections during review, we landed on what we have now at the surrounding fields.

Let's just do things consistently with the rest of the fields in this PR, and create an issue to revisit and update all of the fields afterwards, after a proper discussion? I feel like if we were to change the format - we should do it at once for all of the fields, and this shouldn't be in this PR.